mailbox/api/src/main/java/org/apache/james/mailbox/quota/MaxQuotaManager.java

```diff
@@ -121,10 +123,10 @@
     }
 
     default Optional<QuotaSizeLimit> getMaxStorage(Map<Quota.Scope, QuotaSizeLimit> maxStorageDetails) {
-        return OptionalUtils.or(
-            Optional.ofNullable(maxStorageDetails.get(Quota.Scope.User)),
-            Optional.ofNullable(maxStorageDetails.get(Quota.Scope.Domain)),
-            Optional.ofNullable(maxStorageDetails.get(Quota.Scope.Global)));
+        return Stream.of(Quota.Scope.User, Quota.Scope.Domain, Quota.Scope.Global)
```

Comment:
Maybe in `Quota` class should be convenient method like `allScope()` to make something like `Stream.of(Quota.allScope()`. Is it better readble?